paperqa/agents/env.py

```diff
@@ -111,29 +112,30 @@ def __init__(
         self._summary_llm_model = summary_llm_model
         self._embedding_model = embedding_model
 
-    def make_initial_state_and_tools(self) -> tuple[EnvironmentState, list[Tool]]:
```

Comment:
Note: this does break the interface, but I don't think it will affect anyone, it's practically a private method

Reply:
Oddly enough this was the method I had to overwrite in the server but it’s nbd to swap to this interface

Reply:
Well, let's hope no one else did 😆